shared-helpers/src/views/accounts/AccountCard.tsx

```diff
@@ -2,9 +2,9 @@ import { Heading, HeadingGroup, Icon } from "@bloom-housing/ui-seeds"
 import Card from "@bloom-housing/ui-seeds/src/blocks/Card"
 import React from "react"
 import styles from "./AccountCard.module.scss"
-import { CustomIconMap, CustomIconType } from "../shared/CustomIconMap"
+import { CustomIconMap, CustomIconType } from "./CustomIconMap"
 
-interface AccountCardProps {
+type AccountCardProps = {
```

Comment:
Think this can be changed back since we identified the source of the error was file location